### PR TITLE
Fix Azure Redis Heartbeat aggregation

### PR DESCRIPTION
**Pull request by BzSpi**

### CODE REVIEW

**Review by pdecat — APPROVED** [review]

File: modules/integration_azure-redis/variables.tf [hunk 1]
```diff
@@ -41,7 +41,7 @@ variable "heartbeat_timeframe" {
 variable "heartbeat_aggregation_function" {
   description = "Aggregation function and group by for heartbeat detector (i.e. \".mean(by=['host'])\")"
   type        = string
-  default     = ".mean(by=['shardid'])"
+  default     = ".mean(by=['azure_resource_id', 'shardid'])"
```
Comment (Shr3ps) [inline modules/integration_azure-redis/variables.tf:44]: Why not like others detectors?

```suggestion
  default     = ".mean(by=['shardid', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

**Review by Shr3ps — CHANGES_REQUESTED** [review]

File: modules/integration_azure-redis/variables.tf (hunk 1 above)
Comment (BzSpi) [reply modules/integration_azure-redis/variables.tf:44]: Because those dimensions do not exist :shrug:

Comment (Shr3ps) [reply modules/integration_azure-redis/variables.tf:44]: 😵‍💫

**Review by Shr3ps — APPROVED** [review]

File: modules/integration_azure-redis/variables.tf (hunk 1 above)
Comment (BzSpi) [reply modules/integration_azure-redis/variables.tf:44]: Needs to be double check, dimensions are not displayed but seems available.

Comment (BzSpi) [reply modules/integration_azure-redis/variables.tf]: Changed

**Pull request merged**